src/parachain/amm.ts

```diff
@@ -43,7 +43,9 @@ import {
 } from "..";
 import { StableLiquidityMetaPool } from "./amm/liquidity-pool/stable-meta";
 
-const HOP_LIMIT = 4; // TODO: add as parameter?
+const HOP_LIMIT = 4;
+const FEE_MULTIPLIER_STANDARD = 1000;
```

Comment:
If we have a way to get these consts from the parachain, can we please add that as future issue/task to github?
And if not... let's chat with backend devs to see if that can be exposed in the future.

Either way: Not blocking.

Reply:
Good point, currently these are not exposed, I'm not sure if it's simple to do that or not though, let's chat with the backend guys!